util/ctags.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+#
+# Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+#
+# Usage: ./util/ctags.sh [...arguments for ctags...]
+#
+# This script runs ctags twice. In the first pass, ctags extract macro
+# definitions. readtags that is part of Universal Ctags converts them
+# to ctags options. In the second pass, ctags reads the options and
+# extracts language objects with expanding the macros.
+#
+# Universal Ctags 6.0.0 or higher is assumed.
+#
+: ${CTAGS=ctags}
+: ${READTAGS=readtags}
+
+if ! type "${CTAGS}" > /dev/null; then
+    echo "${CTAGS}: not found" 1>&2
+    exit 1
+fi
+
+if ! "${CTAGS}" --version | grep -q "Universal Ctags"; then
+    echo "${CTAGS}: not Univrsal Ctags" 1>&2
+    exit 1
+fi
+
+if "${CTAGS}" --version | grep -q "Universal Ctags 5.*"; then
+    echo "${CTAGS}: Univrsal Ctags but too old (5)" 1>&2
```

Comment:
Typo: Univrsal -> Universal 

```suggestion
    echo "${CTAGS}: Universal Ctags version is too old (5)" 1>&2
```

Reply:
Added a commit for fixing the typos.